packages/minter-contracts/bin-ts/english_auction_fa2.types.ts

```diff
@@ -0,0 +1,66 @@
+
+import { address, BigMap, int, nat, timestamp } from '@taquito/contract-type-generator';
+
+type Storage = {
+    pauseable_admin?: {
+        admin: address;
+        paused: boolean;
+        pending_admin?: address;
+    };
+    current_id: nat;
+    max_auction_time: nat;
+    max_config_to_start_time: nat;
+    bid_currency: {
+        fa2_address: address;
+        token_id: nat;
+    };
+    auctions: BigMap<nat, {
+        seller: address;
+        current_bid: nat;
+        start_time: timestamp;
+        last_bid_time: timestamp;
+        round_time: int;
+        extend_time: int;
+        asset: Array<{
+            fa2_address: address;
+            fa2_batch: Array<{
+                token_id: nat;
+                amount: nat;
+            }>;
+        }>;
+        min_raise_percent: nat;
+        min_raise: nat;
+        end_time: timestamp;
+        highest_bidder: address;
+    }>;
+};
+
+type Methods = {
+    confirm_admin: () => Promise<void>;
+    pause: (param: boolean) => Promise<void>;
+    set_admin: (param: address) => Promise<void>;
+    bid: (params: {
+        asset_id: nat;
+        bid_amount: nat;
+    }) => Promise<void>;
+    cancel: (param: nat) => Promise<void>;
+    configure: (params: {
+        opening_price: nat;
+        min_raise_percent: nat;
+        min_raise: nat;
+        round_time: nat;
+        extend_time: nat;
+        asset: Array<{
+            fa2_address: address;
+            fa2_batch: Array<{
+                token_id: nat;
+                amount: nat;
+            }>;
+        }>;
+        start_time: timestamp;
+        end_time: timestamp;
+    }) => Promise<void>;
+    resolve: (param: nat) => Promise<void>;
+};
+
+export type EnglishAuctionFa2ContractType = { methods: Methods, storage: Storage, code: { __type: 'EnglishAuctionFa2Code' } };
```

Comment:
Let's update the `code` key of the generated contract types to include the other properties.

```typescript
// ...
code: { __type: 'EnglishAuctionFa2Code'; protocol: string; code: object[] }
```